src/main/java/teammates/ui/controller/StudentCourseDetailsPageData.java

```diff
@@ -5,15 +5,48 @@
 import teammates.common.datatransfer.AccountAttributes;
 import teammates.common.datatransfer.CourseDetailsBundle;
 import teammates.common.datatransfer.InstructorAttributes;
+import teammates.common.datatransfer.StudentAttributes;
 import teammates.common.datatransfer.TeamDetailsBundle;
+import teammates.ui.template.StudentCourseDetailsPanel;
 
 public class StudentCourseDetailsPageData extends PageData {
     public CourseDetailsBundle courseDetails;
-    public TeamDetailsBundle team;
     public List<InstructorAttributes> instructors;
+    public TeamDetailsBundle team;
```

Comment:
Is this used outside the class?
Does this PageData need to keep the whole bundle?